lang/cs/Org.Apache.REEF.IO.Tests/TestAzureBlockBlobFileSystemE2E.cs

```diff
@@ -138,6 +139,54 @@ public void TestDeleteE2E()
             Assert.False(CheckBlobExists(blob));
         }
 
+        [Fact(Skip = SkipMessage)]
+        public void TestGetChildBlobsInContainerE2E()
+        {
+            // setup
+            string[] parentFileNames = new string[] { "sample1", "sample2", "sample3" };
+            string[] folderfileNames = new string[] { "folder1/sample4", "folder1/sample5" };
+            string[] parentListNames = new string[] { "sample1", "sample2", "sample3", "folder1/" };
+
+            foreach (string uploadedBlobName in parentFileNames.Concat(folderfileNames))
+            {
+                CloudBlockBlob blob = _container.GetBlockBlobReference(uploadedBlobName);
+                UploadFromString(blob, "hello");
```

Comment:
Are these files cleaned up anywhere if the test fails and you want to rerun?

Reply:
Yes, the Dispose() method cleans up the container and everything in it.